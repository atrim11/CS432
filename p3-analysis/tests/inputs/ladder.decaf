def
int
main
()
{
    result = 5 * 3 + 12 / (4 - 2) && 10 >= 2 || (7 + 9 * 2 <= 30) && !(15 % 4 == 3) || 8 / 2 - 1 + 6 * (3 + 2) == 25 % 6 && (14 <= 7 || 6 > 3) + (5 * 2 / 10) - (12 == 3) * (4 != 2) / 2 + (7 % 3) * 5 + 9;
    return result;
}
