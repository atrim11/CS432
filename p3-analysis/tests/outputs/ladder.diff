--- outputs/ladder.txt	2024-10-23 17:00:02.826273865 -0400
+++ expected/ladder.txt	2024-10-23 14:21:19.016730767 -0400
@@ -1,3 +1,7 @@
-ERROR: No 'type' attribute
-Error: Variable 'result' used without being defined on line 6
-Error: Variable 'result' used without being defined on line 7
+Symbol 'result' undefined on line 6
+Type mismatch: bool expected but int found on line 6
+Type mismatch: int expected but bool found on line 6
+Type mismatch: int expected but bool found on line 6
+Type mismatch: int expected but bool found on line 6
+Type mismatch: bool expected but int found on line 6
+Symbol 'result' undefined on line 7
