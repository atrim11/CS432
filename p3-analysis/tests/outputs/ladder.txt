ERROR: No 'type' attribute
Error: Variable 'result' used without being defined on line 6
Error: Variable 'result' used without being defined on line 7
