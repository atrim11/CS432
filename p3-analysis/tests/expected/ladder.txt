Symbol 'result' undefined on line 6
Type mismatch: bool expected but int found on line 6
Type mismatch: int expected but bool found on line 6
Type mismatch: int expected but bool found on line 6
Type mismatch: int expected but bool found on line 6
Type mismatch: bool expected but int found on line 6
Symbol 'result' undefined on line 7
